CONTRIBUTING.md

```diff
@@ -1 +1,364 @@
 # Developer Guide
```

Comment:
Do we need to repeat many duplicated contents here, or we just point to CONTRIBUTING.md in antrea for shared contents?

@salv-orlando

Reply:
agreed. In general we should have just pointers to antrea and avoid duplication.

I discussed how to do with @yuntanghsu  - we will mostly have pointers to antrea, with some additional info for items that are specific to Theia.